gammapy/datasets/flux_points.py

```diff
@@ -149,7 +153,17 @@ def write(self, filename, overwrite=False, **kwargs):
 
         table["mask_fit"] = mask_fit
         table["mask_safe"] = self.mask_safe
-        table.write(make_path(filename), overwrite=overwrite, **kwargs)
+
+        filename = make_path(filename)
+
+        if "fits" in filename.suffixes:
+            primary_hdu = fits.PrimaryHDU()
+            hdu_table = fits.BinTableHDU(table, name="TABLE")
+            fits.HDUList([primary_hdu, hdu_table]).writeto(
+                filename, overwrite=overwrite, checksum=checksum
+            )
+        else:
+            table.write(make_path(filename), overwrite=overwrite, **kwargs)
```

Comment:
codecov complain about these lines not being covered?

Reply:
yes, we have nowhere a test to export a fluxpoints dataset to an ascii table. In theory it was possible, but untested. I left this possibility open. But you're right it would need a test to check.